pkg/autoscaler/config/config_test.go

```diff
@@ -70,6 +71,7 @@ func TestNewConfig(t *testing.T) {
 			c.TargetBurstCapacity = 12345
 			c.ContainerConcurrencyTargetDefault = 10.5
 			c.ContainerConcurrencyTargetFraction = 0.71
+			c.TargetUtilization = 0.71
```

Comment:
is there a possibility of `ContainerConcurrencyTargetFraction` and `TargetUtilization` having different values now that both are in the configmap

Reply:
Yes, it is possible. I changed this test case to set different vaule.